app/controllers/saml_controller.rb

```diff
@@ -16,7 +16,11 @@ def login
 
     if response.is_valid? && response.name_id.present?
       username = response.name_id.downcase
```

Comment:
My understanding is response.name_id will hold the keycloak id value when keycloak is ident mgr.  this logic doesn't seem to reflect that?

Reply:
You're right, it belongs in the `else` condition below. Thanks.